src/data/mystery-encounters/utils/encounter-phase-utils.ts

```diff
@@ -547,23 +614,40 @@ export function handleMysteryEncounterVictory(scene: BattleScene, addHealPhase:
   }
 }
 
-export function hideMysteryEncounterIntroVisuals(scene: BattleScene): Promise<boolean> {
+/**
+ *
```

Comment:
text?

Reply:
tweens doesn't interpret numbers properly for some reason